applications/FreeSurfaceApplication/python_scripts/edgebased_levelset_solver.py

```diff
@@ -165,8 +162,7 @@ def PrepareModelPart(self) -> None:
         self.model_part.SetBufferSize(self.GetMinimumBufferSize())
 
     def Check(self) -> None:
-        if all(not component for component in self.body_force):
-            raise ValueError("Body force cannot be a zero vector")
+        pass
```

Comment:
```suggestion
        super().Check()
```
Instead of doing nothing I'd at least call the `Check` of the base solver.

Reply:
Done